docs/crds/gslbhostrule.md

```diff
@@ -26,6 +27,11 @@ spec:
   - cluster: oshift
     weight: 5
     priority: 10
+   publicIP:
+  - cluster: k8s
+    ip: 160.10.1.1   # or 2345:0425:2CA1:0:0:0567:5673:23b5
```

Comment:
Have we tested the IPv6? If not please remove this comment and mention that we only support ipv4 presently.

Reply:
Removed the comment since IPv6 is only partially tested and not at all by system test.